eta/core/video.py

```diff
@@ -738,12 +738,19 @@ def get_frame_range(self):
         fns = self.get_frame_numbers()
         return (fns[0], fns[-1]) if fns else (None, None)
 
-    def merge_video_labels(self, video_labels):
+    def merge_video_labels(self, video_labels, reindex_objs=False):
         '''Merges the given VideoLabels into this labels.
 
         Args:
             video_labels: a VideoLabels instance
+            reindex_objs: whether to reindex objects in `self` before merging
```

Comment:
small thing, but it seems the more natural thing to do would be to change the `index`es of the *new objects*, not the ones in `self`

Reply:
but that would mutate the input `VideoLabels`, whereas `self` is going to be changed internally anyways due to merging.